### PR TITLE
Eg/implement timer for group card

### PR DESCRIPTION
**Pull request by Ezex360**
timers

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,6 +28,7 @@
     "components": "^0.1.0",
     "graphql": "^14.6.0",
     "graphql-tag": "^2.10.3",
+    "moment": "^2.24.0",
```
Comment (abelosorio) [inline package.json:31]: Let's use dayJS instead. It's more performant.

File: src/components/GroupCard/index.js [hunk 2]
```diff
@@ -27,6 +29,11 @@ const GroupCard = ({
 
   return (
     <div className='group-card' onClick={() => goToCheckout()}>
+      <Timer
+        expiresAt={expiresAt}
+        bottomPosition='94%'
```
Comment (abelosorio) [inline src/components/GroupCard/index.js:34]: Shouldn't these options go into CSS?

File: src/components/Timer/index.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+import './timer.scss'
+import TimerIcon from '../../vectors/timer.svg'
+
+const Timer = ({
+  expiresAt, topPosition = 0, rightPosition = 0, bottomPosition = 0, leftPosition = 0
+}) => {
+  const expireDate = Date.parse(expiresAt)
+
+  const calculateTimeLeft = () => {
+    const difference = +expireDate - +new Date()
```
Comment (abelosorio) [inline src/components/Timer/index.js:12]: Why these plus signs? It's not very clear. Also, you can calculate this much easier using dayjs (or moment)'s operators.

File: src/components/Timer/timer.scss [hunk 4]
```diff
@@ -0,0 +1,23 @@
+.Timer{
```
Comment (abelosorio) [inline src/components/Timer/timer.scss:1]: Missing space here.

File: src/components/Timer/index.js (hunk 3 above)
Comment (Ezex360) [reply src/components/Timer/index.js:12]: The plus signs is a sintax sugar to parse to Integer from any other type. I wasn't trying to use any library, it must be an auto-import from a previous attempt, I will be more carefull next time.
And I will search for dayJs and see how to use it here

File: src/components/GroupCard/index.js (hunk 2 above)
Comment (Ezex360) [reply src/components/GroupCard/index.js:34]: These were there to be able to change the position from arguments like the DiscountBandages module, but if you say that these are not necessary I can move them to the .scss file

**Pull request merged**